lib/kubernetes-deploy/runner_task.rb

```diff
@@ -122,6 +122,7 @@ def watch_pod(pod)
 
     def record_status_once(pod)
       cache = ResourceCache.new(@task_config)
+      cache.prewarm([pod])
```

Comment:
I guess it doesn't hurt either, but prewarming won't change anything with a single resource.

Reply:
Fair! I'll clean this up.